VCFv4.3.tex

```diff
@@ -1602,7 +1604,7 @@ \subsubsection{Type encoding}
 \vspace{0.3cm}
 
 Note this is not used in BCF2, but its type is reserved in case this becomes necessary.
-In BCF2 characters are simply represented by strings with a single element 0,4,6,8--15 reserved for future use.
+In BCF2 characters are simply represented by strings with a single element 4,6,8--15 reserved for future use.
```

Comment:
What does this paragraph even mean? What is “this” that's not used in BCF2? It's been like this since the document was first added (8b0a59271f13eea9cff30c6e372b7fe675f815f0). 

 Since you're changing it please fix it. It looks like it's referring to something that doesn't exist any more. So perhaps this paragraph can just become something to the effect of:

> Something about strings are type=7 of the appropriate length, and characters are simply represented as length=1 strings. Type codes 4,6,8--15 are reserved for future use.

First sentence optional; note fullstop added before `4,6,…`.

And similarly in the other two documents.

Reply:
To be precise: The two separate ideas and sentences run together as “_In BCF2 characters are simply represented by strings with a single element 0,4,6,8--15 reserved for future use_” is a historical typographical error and **needs** to be fixed. The previous sentence (“_Note this is not used in BCF2, but its type is reserved in case this becomes necessary_”) is a mystery that it **would be nice** to fix while working in the area, perhaps by simply removing it.